MoltenVK/MoltenVK/Vulkan/mvk_datatypes.mm

```diff
@@ -608,8 +608,13 @@ MVK_PUBLIC_SYMBOL MTLStoreAction mvkMTLStoreActionFromVkAttachmentStoreOp(VkAtta
 // If we need to resolve, but the format doesn't support it, we must store the attachment so we can run a post-renderpass compute shader to perform the resolve.
 MTLStoreAction mvkMTLStoreActionFromVkAttachmentStoreOpInObj(VkAttachmentStoreOp vkStoreOp, bool hasResolveAttachment, bool canResolveFormat, MVKBaseObject* mvkObj) {
 	switch (vkStoreOp) {
-		case VK_ATTACHMENT_STORE_OP_STORE:		return hasResolveAttachment && canResolveFormat ? MTLStoreActionStoreAndMultisampleResolve : MTLStoreActionStore;
-		case VK_ATTACHMENT_STORE_OP_DONT_CARE:	return hasResolveAttachment ? (canResolveFormat ? MTLStoreActionMultisampleResolve : MTLStoreActionStore) : MTLStoreActionDontCare;
+		// Metal does not support VK_ATTACHMENT_STORE_OP_NONE. Next best option is store.
```

Comment:
Sorry, I just saw this. From the Vulkan spec:
```
VK_ATTACHMENT_STORE_OP_NONE specifies the contents within the render area are not accessed by the store operation as long as no values are written to the attachment during the render pass. If values are written during the render pass, this behaves identically to VK_ATTACHMENT_STORE_OP_DONT_CARE and with matching access semantics.
```

So unsure if storing here is the best option, shouldn't it be the same as `VK_ATTACHMENT_STORE_OP_DONT_CARE`?

Reply:
Have a look at the discussion mentioned above in https://github.com/KhronosGroup/MoltenVK/issues/1930#issuecomment-2089164911, and let me know what your thoughts are after that.

It's a grey area, and the question is about what is the state of the attachment for future render passes.

Reply:
It does make sense for the canonical render pass approach since the attachment is only supposed to be written at the end, but contents are preserved throughout the subpasses.

For the dynamic rendering approach, it would not be the optimal approach since we would be paying the cost of the actual storage when it cannot be used as input attachment (maybe with `VK_KHR_dynamic_rendering_local_read` extension can). But I guess we can live with it unless there's some CTS test that complains about this I guess (unlikely).